app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Again here we first need to establish the association between the User and the Post correctly. We need to change the `belongs_to :user` line to 
  
 `belongs_to :author, class_name: 'User'`

- This line establishes an association between the "Post" model and the "User" model. It creates a foreign key in the "posts" table called "author_id," which references the "users" table.
- The class_name option specifies the name of the associated class (in this case, 'User').